multicluster/apis/multicluster/v1alpha1/clusterset_types.go

```diff
@@ -27,14 +27,23 @@ type MemberCluster struct {
 	ClusterID string `json:"clusterID,omitempty"`
 	// API server of the destination cluster.
 	Server string `json:"server,omitempty"`
-	// Secret name to access API server of the member from the leader cluster.
+	// Secret name in the member cluster which will be used to access API
```

Comment:
Name of the Secret resource in the member cluster, which stores the token to access the leader cluster's API server.